### PR TITLE
added commas to badges output

### PR DESCRIPTION
**Pull request by ish-u**
Hi ,
I don't know PHP that well.
This is my first Pull Request.

### CODE REVIEW

File: web/subscribers/index.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "style" => "flat-square",
     "label" => "YouTube subscribers",
     "labelColor" => "gray",
-    "id" => "UCipSxT7a3rn81vGLw9lqRkg",
+    "id" => "UC-lHJZR3Gqxm24_Vd_AJ5Yw",
```
Comment (DenverCoder1) [inline web/subscribers/index.php]: PewDiePie's great, but I'd like to keep the default ID as is.

File: web/subscribers/index.php [hunk 2]
```diff
@@ -24,4 +24,10 @@
 header('Content-type: image/svg+xml');
 
 // Get response from the URL and output its contents
-echo curl_get_contents($url);
\ No newline at end of file
+$response = (curl_get_contents($url));
+
+//Adding Commas 
+preg_match_all('!\d+!',strip_tags($response),$matches);
+$response =  str_replace($matches[0][0],number_format($matches[0][0]),$response);
```
Comment (DenverCoder1) [inline web/subscribers/index.php]: Improvements to code formatting:

* the parentheses around the curl_get_contents function are not necessary
* add spaces after commas separating the function parameters

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution. 👍

This is a good first pull request.

Things to note:
* It's recommended when fixing an issue to reference it like this: Fixes #1
* Another user had already requested to work on this issue when you submitted your pull request. It's generally best to avoid doing that. In this particular case, I decided to allow it since it seemed no progress had been made and there were no comments for over a day.
* Before you start working on an issue, it's a good idea to comment on it, so that everyone is aware that it is being worked on and the issue can be discussed.

If you could look over the attached review and add these changes to your commit, that would be great! Let me know when you have made the changes and I can merge.

**Comment by ish-u** [comment]
hi,
I have done the necessary chaanges you requested.
Thanks for the tips, I will remember to comment before start working on issues from next time.

**Review by DenverCoder1 — APPROVED** [review]
Thanks, looks good!

**Pull request merged**